### DISCUSSION TITLE
How can I disable the separator line completely?

### DISCUSSION THREAD

**Discussion by subhoghoshX** [CopilotC-Nvim/CopilotChat.nvim#367] [Q&A] [answered]
I'm talking about the white line after ##User:

![image](https://github.com/user-attachments/assets/633502b0-77cf-4d50-bef6-38ccac00a249)

Often, when I undo or make some edits, it comes down like this, hiding the text below it:

![image](https://github.com/user-attachments/assets/0086b4b0-cf3f-4396-84b9-6aad6889ea07)

Setting the `separator` option to empty space doesn't work either; instead of the line character, now the empty space character hides the text.

```lua
require('CopilotChat').setup({
  separator = ' ',  -- this doesn't solve the problem
})
```

I appreciate any help. And a huge thanks for building this plugin!

**Comment by craigrosie** [comment]
Agreed on this, I would prefer to be able to disable it completely

**Comment by deathbeam** [comment, marked as answer]
Disable `highlight_headers` config option, the separator needs to be there for parsing, but that single line comes from extmark from `highlight_headers` option
